cmd/root.go

```diff
@@ -64,17 +75,36 @@ func init() {
 }
 
 func initConfig() {
+
+	// As Viper cannot unmarshal and merge configs from yaml automatically,
+	// define inline there the mapping explictly.
+	// See: https://github.com/spf13/viper/issues/761
+	viper.SetDefault("NAMESPACE", "")
+	viper.SetDefault("LOGGREGATOR_KEY_PATH", "")
+	viper.SetDefault("LOGGREGATOR_ENDPOINT", "")
+	viper.SetDefault("LOGGREGATOR_CA_PATH", "")
+	viper.SetDefault("LOGGREGATOR_CERT_PATH", "")
```

Comment:
I don't think we actually need these (since they're the zero values already).
```suggestion
```

Reply:
we need that, otherwise you cannot set them up properly without providing a config :man_shrugging: